content/en/docs/additional-concepts/hpa/_index.md

```diff
@@ -0,0 +1,106 @@
+deployment.apps/hpa-demo-deployment   1/1     1            1           42h
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+pod/hpa-demo-deployment-9cc6d54b5-kprvn   1/1     Running   0          42h
+
+NAME                                                      REFERENCE                        TARGETS       MINPODS   MAXPODS   REPLICAS   AGE
+horizontalpodautoscaler.autoscaling/hpa-demo-deployment   Deployment/hpa-demo-deployment   cpu: 6%/50%   1         10        1          42h
+```
+
+And then you see that new Pods are being scheduled in our Namespcase, because the current CPU utilization is at around 250%. The HPA will now scale your deployment until it reaches again the target 50% CPU utilization or when MAXPODS is reached:
+
+```
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/hpa-demo-deployment   6/6     6            6           42h
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+pod/hpa-demo-deployment-9cc6d54b5-7fnkf   1/1     Running   0          11s
+pod/hpa-demo-deployment-9cc6d54b5-k5tdg   1/1     Running   0          26s
+pod/hpa-demo-deployment-9cc6d54b5-kgngq   1/1     Running   0          26s
+pod/hpa-demo-deployment-9cc6d54b5-kprvn   1/1     Running   0          42h
+pod/hpa-demo-deployment-9cc6d54b5-t9xhq   1/1     Running   0          26s
+pod/hpa-demo-deployment-9cc6d54b5-vt9zg   1/1     Running   0          11s
+
+NAME                                                      REFERENCE                        TARGETS         MINPODS   MAXPODS   REPLICAS   AGE
+horizontalpodautoscaler.autoscaling/hpa-demo-deployment   Deployment/hpa-demo-deployment   cpu: 249%/50%   1         10        4          42h
+```
+
+Once the CPU utilization reaches around 50% again, no more new pods will be created and the replica count remains on that level:
+
+```
+NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/hpa-demo-deployment   6/6     6            6           42h
+
+NAME                                      READY   STATUS    RESTARTS   AGE
+pod/hpa-demo-deployment-9cc6d54b5-7fnkf   1/1     Running   0          39s
+pod/hpa-demo-deployment-9cc6d54b5-k5tdg   1/1     Running   0          54s
+pod/hpa-demo-deployment-9cc6d54b5-kgngq   1/1     Running   0          54s
+pod/hpa-demo-deployment-9cc6d54b5-kprvn   1/1     Running   0          42h
+pod/hpa-demo-deployment-9cc6d54b5-t9xhq   1/1     Running   0          54s
+pod/hpa-demo-deployment-9cc6d54b5-vt9zg   1/1     Running   0          39s
+
+NAME                                                      REFERENCE                        TARGETS        MINPODS   MAXPODS   REPLICAS   AGE
+horizontalpodautoscaler.autoscaling/hpa-demo-deployment   Deployment/hpa-demo-deployment   cpu: 46%/50%   1         10        6          42h
+```
+When you stop the `load-generator`you also see that the deployment scales back to 1 replica:
```

Comment:
Stop the `load-generator` by closing the terminal. You will see that the deployment scales back to 1 replica.